mache-activemq/build.gradle

```diff
@@ -5,4 +5,5 @@ dependencies {
   testCompile 'org.mockito:mockito-all:1.10.19'
   testCompile project(path: ':mache-observable', configuration: 'tests')
   testCompile project(path: ':mache-core', configuration: 'tests')
+  testCompile 'org.mockito:mockito-all:1.10.19'
```

Comment:
another duplicat "testCompile 'org.mockito:mockito-all:1.10.19'" - i reckon this can go in the main build.gradle - it pretty much goes hand in hand with junit which is also set up there.